src/table/pagination-table/components/body/withStyling.tsx

```diff
@@ -26,6 +26,7 @@ export default function withStyling(CellComponent: typeof StyledBodyCell) {
         align={align}
         tabIndex={tabIndex}
         className={`sn-table-cell ${selectedCellClass || ''}`}
+        title={(children as ReactElement).props.children}
```

Comment:
I don't really understand this one. Care to explain?

Reply:
children here is a StyledBodyCell,which has the text we want as a tooltip in its props, but children is of type ReactNode by default, which doesn't always have props.
So I'm type casting it to a ReactElement

Reply:
Ah now I see. So `children.props.children` turns out to the text node.

I do think this could break rather easily. A change in the depth of the text node and this will break.

I would consider a different solution. For example just passing the title as a prop the cell renderer?

Reply:
passing the title as a prop might work. If we add it on CellText, it will be only on the text itself and not the cell, and yeah, it might not work at all in the header.
I'll look into it some more :)